d1985r0.md

```diff
@@ -428,58 +427,88 @@ template<typename T, typename A> my_container(T*, size_t, const A&) -> my_contai
 
 As the primary template is not defined there is only a default
 constructor implicit deduction guide. The explicit deduction guides
-select the first specialization unless an allocator object is given
-in which case the second specialization is selected. To select the
-third specialization the template arguments must be explicitly given.
+select the first specialization unless an allocator object is given in
+which case the second specialization is selected. To select the third
+specialization the template arguments must be explicitly given.
 
 
-## Overloading of variable templates
+## Impacts on the specialization of variable templates
 
-Universal template parameters can be used to implement what appears like overloading a variable
-template. The problem of not being able to delete the base case then becomes more
-pressing than currently as selecting the base case should trigger an error but
-will be unconditionally selected when none of the specialziations matches. The dependent
-static_assert mentioned above can be used to rescue this situation but
-this is still somewhat of a hack.
+Universal template parameters can be used to implement what appears
+like overloading of variable templates. 
+
+The problem of not being able to delete the base case then becomes
+more pressing than currently as selecting the base case should trigger
```

Comment:
s/more pressing than currently/even more pressing than in c++20/, also add comma before "as"